dynawo/sources/Modeler/Common/DYNDelayManager.h

```diff
@@ -144,28 +146,31 @@ class DelayManager {
   }
 
   /**
-   * @brief End triggers
-   *
-   * This will reset currently activated triggers
-   */
-  void notifyEndTrigger();
-
- private:
-  /**
-   * @brief Retrieves the delay by id (const version)
-   *
-   * Precondition: the id is acceptable
-   *
-   * @param id the id to use
-   * @returns the corresponding delay
-   */
+  * @brief Retrieves the delay by id (const version)
+  *
+  * Precondition: the id is acceptable
+  *
+  * @param id the id to use
+  * @returns the corresponding delay
+  */
   const Delay& getDelayById(size_t id) const {
     return delays_.at(id);
   }
 
+  /**
+  * @brief Retrieves the delay by id (non-const version)
+  *
+  * Precondition: the id is acceptable
+  *
+  * @param id the id to use
+  * @returns the corresponding delay
+  */
+  Delay& getNonCstDelayById(size_t id) const {
```

Comment:
There should be a setter in Delay manager

Reply:
You mean a setDelayTime ?

Reply:
yes